protocol/contracts/beanstalk/AppStorage.sol

```diff
@@ -241,14 +241,16 @@ contract Storage {
     /**
      * @notice System-level Silo state; contains deposit and withdrawal data for a particular whitelisted Token.
      * @param deposited The total amount of this Token currently Deposited in the Silo.
+     * @param bdv The total bdv of this Token currently Deposited in the Silo.
```

Comment:
should this be `depositedBdv`?

Reply:
See that you fixed it here:
https://github.com/BeanstalkFarms/Beanstalk/pull/457/commits/09ecaa347562ef4916399cb95c9d66ff52ebad55
ty